public/controllers/dev-tools/dev-tools.js

```diff
@@ -232,8 +238,33 @@ export class DevToolsController {
     return affectedGroups;
   }
 
+  async getAvailableMethods() {
+    try {
+      const response = await this.genericReq.request('GET', '/api/routes', {});
+      this.apiInputBox.model = !response.error ? response.data : [];
+    } catch (error) {
+      this.apiInputBox.model = [];
+    }
+  }
+
   init() {
     this.apiInputBox.setSize('auto', '100%');
+    const ExcludedIntelliSenseTriggerKeys = {
```

Comment:
Please move this to an external file:

```js
export const excludedIntelliSenseTriggerKeys = {
...
}
```

Then in top of this controller use:

```js
import { excludedIntelliSenseTriggerKeys } from './excluded-keys.js'
```